specification/cognitiveservices/data-plane/AutoSuggest/readme.enable-multi-api.md

```diff
@@ -0,0 +1,23 @@
+# Multi-API support for AutoRest v3 generators
```

Comment:
What the heck happened here??

Reply:
I didn't realize that `input_file` is `string[]` or `string`. Fixed.